polaris.shopify.com/pages/api/v0/tokens/[tokens].tsx

```diff
@@ -97,67 +58,47 @@ const formatTokenGroup = (tokenGroup: TokenGroup, format: Format) => {
       .join('\n');
   }
 
-  return tokenGroup;
+  return tokenValues;
 };
 
 const handler = async (req: NextApiRequest, res: NextApiResponse) => {
   const formatParam = (req.query.format || 'json') as Format;
```

Comment:
```suggestion
  const formatParam = isFormat(req.query.format) ? req.query.format : 'json';
```

We already have a `isFormat` type guard. Perhaps, we should use that here so users can't define an arbitrary format which TypeScript is currently casting to the `Format` type. Thoughts?